website/server/controllers/api-v3/challenges.js

```diff
@@ -26,13 +26,125 @@ let api = {};
+ * @apiSuccess {String} challenge._id Same as `challenge.id`.
+ * @apiSuccess {String} challenge.prize Number of gems offered as prize to winner.
+ * @apiSuccess {String} challenge.memberCount Number users participating in challenge.
+ * @apiSuccess {Object} challenge.tasksOrder Object representing associated tasks and rewards.
+ * @apiSuccess {Array} challenge.tasksOrder.rewards Array of `reward` objects associated with challenge.
+ * @apiSuccess {Array} challenge.tasksOrder.todos Array of `todo` objects associated with challenge.
+ * @apiSuccess {Array} challenge.tasksOrder.dailys Array of `dailys` objects associated with challenge.
+ * @apiSuccess {Array} challenge.tasksOrder.habits Array of `habits` objects associated with challenge.
+ * @apiSuccess {Boolean} challenge.official Boolean indicating if this is an official Habitica challenge.
+ *
+ */
+
+/**
+ * @apiDefine ChallengeSuccessExample
+ * @apiSuccessExample {json} Sucessfull response with single challenge
+ {
+   "data": {
+     "group": {
+      "_id": "group-id-associated-with-challenge",
+      "name": "MyGroup",
+      "type": "guild",
+      "privacy": "public"
+     },
+     "name": "Long Detailed Name of Challenge",
+     "shortName": "my challenge",
+     "leader": {
+       "_id": "user-id-of-challenge-creator",
+       "profile": {
+         "name": "MyUserName"
+       }
+     },
+     "updatedAt": "timestamp,
+     "createdAt": "timestamp",
+     "_id": "challenge-id",
+     "prize": 0,
+     "memberCount": 1,
+     "tasksOrder": {
+       "rewards": [],
+       "todos": [],
+       "dailys": [],
+       "habits": []
+     },
+     "official": false,
+     "id": "challenge-id"
+   }
+ }
+*/
+
+/**
+ * @apiDefine ChallengeArrayExample
+ * @apiSuccessExample {json} Sucessfull response with array of challenges
+ {
+   "data": [{
+     "group": {
+       "_id": "group-id-associated-with-challenge",
+       "name": "MyGroup",
+       "type": "guild",
+       "privacy": "public"
+     },
+     "name": "Long Detailed Name of Challenge",
+     "shortName": "my challenge",
+     "leader": {
+       "_id": "user-id-of-challenge-creator",
+       "profile": {
+         "name": "MyUserName"
+       }
+     },
+     "updatedAt": "timestamp,
+     "createdAt": "timestamp",
+     "_id": "challenge-id",
+     "prize": 0,
+     "memberCount": 1,
+     "tasksOrder": {
+       "rewards": [],
+       "todos": [],
+       "dailys": [],
+       "habits": []
+     },
+     "official": false,
+     "id": "challenge-id"
+   }]
+ }
+*/
+
+/**
  * @api {post} /api/v3/challenges Create a new challenge
  * @apiName CreateChallenge
  * @apiGroup Challenge
+ * @apiDescription Creates a challenge. Cannot create associated tasks with this route.
+ *
+ * @apiParam (Body) {Object} challenge An object representing the challenge to be created
+ * @apiParam (Body) {String} challenge.groupId The id of the group to receive the challenge.
```

Comment:
`to receive the challenge` doesn't sound quite right to me. Maybe `The id of the group to which the challenge belongs.`

 Also, the type should be `{UUID}` instead of `{STRING}`